src/client/renderer/r_types.h

```diff
@@ -275,12 +374,11 @@ typedef struct {
 } r_bsp_texinfo_t;
 
 typedef struct {
-	vec3_t origin;
 	vec_t radius;
-	const r_image_t *image;
-	vec3_t color;
 	uint32_t time;
 	vec_t alpha;
+
+	r_particle_t particle;
 } r_bsp_flare_t;
```

Comment:
Is it possible to remove the flare's alpha and timestamps, and just use those of the backing particle?